### PR TITLE
refactor: rename 'ToolchainEvent' type to 'SocialEvent'

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -34,4 +34,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220405142115-a332d4046e57
```
Comment (alexeykazakov) [inline go.mod:37]: Reminder.

**Review by alexeykazakov — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (xcoulon) [reply go.mod]: addressed in [a04ef52](https://github.com/codeready-toolchain/toolchain-e2e/pull/531/commits/a04ef52075a850d4405faf97bb5a3a9157aa2ee5)

**Review by MatousJobanek — APPROVED** [review]

**Pull request merged**